### PR TITLE
Fix several notices

### PR DESCRIPTION
**Pull request by MrPetovan**
Address:
- https://github.com/friendica/friendica/issues/9252#issuecomment-752730033
- https://github.com/friendica/friendica/issues/10169#issuecomment-827182936
- https://github.com/friendica/friendica/issues/10169#issuecomment-835546485
- https://github.com/friendica/friendica/issues/10169#issuecomment-835547443
- https://github.com/friendica/friendica/issues/10169#issuecomment-844324800

### CODE REVIEW

File: src/Content/Item.php [hunk 1]
```diff
@@ -127,16 +127,6 @@ public static function replaceTag(&$body, &$inform, $profile_uid, $tag, $network
 			$tag_type = substr($tag, 0, 1);
 			//is it already replaced?
 			if (strpos($tag, '[url=')) {
-				// Checking for the alias that is used for OStatus
```
Comment (annando) [inline src/Content/Item.php:130]: Why is this removed?

Comment (MrPetovan) [reply src/Content/Item.php:130]: Assigned on line 136, `$newtag` isn't used because of the early `return` statement on line 140. As a result, the `$data` assignment on line 133 also is superfluous and the `preg_match` on line 132 is as well, and finally the comment doesn't seem to reference anything anymore.
